exercises/forth/example.py

```diff
@@ -1,8 +1,12 @@
+class UnderflowError(Exception):
```

Comment:
Minor point, but I think this might be better as a `StackUnderflowError` to make its meaning a bit more clear and to differentiate it from arithmetic underflow.

Reply:
When you put it that way, I agree. Original intent was to match convention, since there is an existing OverflowError.